parsers/config.py

```diff
@@ -199,6 +201,9 @@
     # 'MD->RO':     ENTSOE.fetch_exchange,
     # MK
     'MK->RS':     ENTSOE.fetch_exchange,
+    # NI
+    'CR->NI':     NI.fetch_exchange,
```

Comment:
Ah sure, I noticed that trend when adding it to web app exchanges config, but forgot it here. Will fix.